Neos.ContentRepositoryRegistry/Tests/Unit/Service/CatchUpDeduplicationQueueTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+namespace Neos\ContentRepositoryRegistry\Tests\Unit\Service;
+
+use Neos\ContentRepository\Core\Factory\ContentRepositoryId;
+use Neos\ContentRepository\Core\Projection\Projections;
+use Neos\ContentRepositoryRegistry\Service\CatchUpDeduplicationQueue;
+use Neos\ContentRepositoryRegistry\Tests\Unit\Service\Fixture\FakeFileCatchUpTrigger;
+use Neos\ContentRepositoryRegistry\Tests\Unit\Service\Fixture\FakeProjectionWithState;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * This is not a regular unit test, it won't run with the rest of the testsuite.
+ * the following two commands after another would run the parallel tests and then the validation of results:
+ * requires "brianium/paratest": "^6.11"
```

Comment:
i see because the latest para test latest requires phpunit 10 but flow blocks that.